ipn/ipnlocal/local.go

```diff
@@ -1969,16 +1964,23 @@ func (b *LocalBackend) setPrefsLockedOnEntry(caller string, newp *ipn.Prefs) {
 		cc.Login(nil, controlclient.LoginDefault)
 	}
 
+	if oldp.RunSSH && oldp.WantRunning {
```

Comment:
if you add a method like `ShouldSSHBeUp()` on the prefs, then this is just:

````go
    if oldp.ShouldSSHBeUp() && !newp.ShouldSSHBeUp() {
     
````

.... which reads a bit nicer

Reply:
Done